scripts/runlogwatch.sh

```diff
@@ -1,20 +1,24 @@
+
```

Comment:
I think this newline might be the reason the CI tests are failing

Looking at the CI job logs, I noticed this to be failing
`failed: (1.3s) 2021-06-18T10:05:50 "[sig-installer][Feature:baremetal] Baremetal platform should have a metal3 deployment [Suite:openshift/conformance/parallel]"
`

which fails if the replica set for the metal3 pod isn't 1
https://github.com/openshift/origin/blob/d389c2654/test/extended/baremetal/hosts.go#L72-L74

Looking at the must-gather, I could see that the "ironic-deploy-ramdisk-logs" was restarted 27 times, the logs for this container have an error
```
$ cat metal3-5444948969-w98zp/ironic-deploy-ramdisk-logs/ironic-deploy-ramdisk-logs/logs/current.log
2021-06-18T10:50:25.394447661Z standard_init_linux.go:219: exec user process caused: exec format error

```

Reply:
oh snap, I didn't see that! thanks!